### DISCUSSION TITLE
Conditional fields - Extend actions

### DISCUSSION THREAD

**Discussion by AlexVanK** [directus/directus#10199] [Feature Requests]
Hello,

in the workflow of my application, I have conditional field B which depends on field A.
A = dropdown.
B = radio

As you can see when the status is "pending", the review radio is disabled:
![image](https://user-images.githubusercontent.com/56647386/144276250-f43075a4-051e-4864-a8d4-b9178ef01f4a.png)

And when the status is "draft", the review radio is enabled:
![image](https://user-images.githubusercontent.com/56647386/144276342-f0a390af-5ad0-4334-aa92-308b99469472.png)

This works fine, but it's a bit faulty, as I need to force a page SAVE then refresh to enable the radio.
Otherwise a user could "temporarily" change the status of A, update B, and then change A back with the new value of B being saved.
We don't want this, this defeats the purpose of the condition...

In other words, I need the radio to be enabled only if the status of the fetched item is on "draft", not if the status changes while editing it.

--- 

At the same time, we need an option to reset the field when the condition triggers:
![image](https://user-images.githubusercontent.com/56647386/144277708-5bc8cb5c-51d9-497d-8293-191107200501.png)

**Comment by rijkvanzanten** [comment]
> Otherwise a user could "temporarily" change the status of A, update B, and then change A back with the new value of B being saved.
> We don't want this, this defeats the purpose of the condition...

Fair enough! I think we could reset the value once the field becomes disabled by default 👍🏻 Wanna open a PR for that?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
